src/pybind/mgr/rook/module.py

```diff
@@ -134,6 +134,25 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+class RookEnv(object):
+    def __init__(self):
+        # POD_NAMESPACE already exist for Rook 0.9
+        pod_namespace = os.environ.get('POD_NAMESPACE', 'rook-ceph')
+        self.cluster_ns = os.environ.get('ROOK_CLUSTER_NS', pod_namespace)
+
+        # ROOK_CLUSTER_NAME was a previously used env var name.
+        rook_cluster_name = os.environ.get('ROOK_CLUSTER_NAME', pod_namespace)
+        # ROOK_CEPH_CLUSTER_CRD_NAME is new is Rook 1.0
+        self.cluster_name = os.environ.get('ROOK_CEPH_CLUSTER_CRD_NAME', rook_cluster_name)
+
+        self.operator_ns = os.environ.get('ROOK_OPERATOR_NAMESPACE', "rook-ceph-system")
```

Comment:
Nit: These should probably be noted as new in 1.0 if the above are also noted as such.

Reply:
yep, but I'm not introducing any changes in a backport PR. All changes will go through the usual PR process in master first.

Reply:
can _ns be renamed to _namespace, since it confusing with nameserver

Reply:
@kshtsk I'd like to not introduce any non-critical changes in this PR

Reply:
See #27556 for renaming the vars